docs/standard/datetime/how-to-use-dateonly-timeonly.md

```diff
@@ -0,0 +1,143 @@
+---
+title: How to use DateOnly and TimeOnly
+description: Learn about the DateOnly and TimeOnly structures. DateOnly and TimeOnly 
+ms.date: 12/28/2022
+dev_langs: 
+  - "csharp"
+  - "vb"
+helpviewer_keywords: 
+  - "DateOnly structure"
+  - "TimeOnly structure"
+  - "date and time classes [.NET]"
+---
+
+# How to use the DateOnly and TimeOnly structures
+
+The <xref:System.DateOnly> and <xref:System.TimeOnly> structures were introduced with .NET 6, and represent either a specific date and time-of-day, respectively. Prior to .NET 6, and always in .NET Framework, the <xref:System.DateTime> type (or some other alternative) was used to represent one of the following:
+
+- A whole date and time.
+- A date, disregarding the time.
+- A time, disregarding the date.
+
+`DateOnly` and `TimeOnly` are types that represent those particular portions of a `DateTime` type.
+
+> [!IMPORTANT]
```

Comment:
Shouldn't this just be a NOTE?

Reply:
IMPORTANT definition says "critical for success" which in this case, comparing .NET latest vs .NET Framework which users may still be using, I wanted to really promote that this is ONLY in .NET latest.